_config.yml

```diff
@@ -19,7 +19,8 @@ github:
 network:
   linkedin: https://www.linkedin.com/company/womencodingcommunity
   slack: https://join.slack.com/t/womencodingcommunity/shared_invite/zt-2hpjwpx7l-rgceYBIWp6pCiwc0hVsX8A
-  github: https://github.com/WomenCodingCommunity
+  github: https://github.com/Women-Coding-Community
+  meetup: https://www.meetup.com/women-coding-community/
```

Comment:
I think we can add youtube: https://www.youtube.com/channel/UCCm36ZZNM8DJPJJGppx9MQw 

It is empty. But soon we are going to add the video for the last in-person event.